pkg/reconciler/kubernetes/tektondashboard/controller.go

```diff
@@ -59,29 +57,65 @@ func NewExtendedController(generator common.ExtensionGenerator) injection.Contro
 			logger.Fatalw("Error creating client from injected config", zap.Error(err))
 		}
 		mflogger := zapr.NewLogger(logger.Named("manifestival").Desugar())
-		manifest, err := mf.ManifestFrom(mf.Slice{}, mf.UseClient(mfclient), mf.UseLogger(mflogger))
+
+		readonlyManifest, err := mf.ManifestFrom(mf.Slice{}, mf.UseClient(mfclient), mf.UseLogger(mflogger))
+		if err != nil {
+			logger.Fatalw("Error creating initial manifest", zap.Error(err))
+		}
+
+		// Reads the source manifest from kodata while initializing the contoller
+		if err := fetchSourceReadOnlyManifests(context.TODO(), &readonlyManifest); err != nil {
+			logger.Fatalw("failed to read manifest", err)
+		}
+
+		fullaccessManifest, err := mf.ManifestFrom(mf.Slice{}, mf.UseClient(mfclient), mf.UseLogger(mflogger))
 		if err != nil {
 			logger.Fatalw("Error creating initial manifest", zap.Error(err))
 		}
 
+		// Reads the source manifest from kodata while initializing the contoller
+		if err := fetchSourceFullAccessManifests(context.TODO(), &fullaccessManifest); err != nil {
```

Comment:
```suggestion
		if err := fetchSourceFullAccessManifests(ctx, &fullaccessManifest); err != nil {
```

Reply:
https://github.com/tektoncd/operator/pull/526
is it possible to use initController for dashboard?

Reply:
yes that  can be done but I thought I'll submit a follow up patch for it

Reply:
how much of the existing initController will have to change to accomodate dashboard 🧑‍💻